install/kubernetes/helm/Istio/charts/initializer/templates/istio-initializer.yaml

```diff
@@ -0,0 +1,86 @@
+{{- $istio_namespace := .Values.global.namespace }}
+{{- $pilot_hub := .Values.global.pilot_hub }}
+{{- $pilot_tag := .Values.global.pilot_tag }}
+################################
+# Istio initializer
+################################
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-inject
+  namespace: {{ $istio_namespace }}
+data:
+  config: |-
+    policy: "enabled"
+    namespaces: [""] # everything, aka v1.NamepsaceAll, aka cluster-wide
```

Comment:
How about make this as parameters in `values.yaml`? This can make the chart can also handle cases of `excludeNamespaces`.